plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/ProjectContextMenuTest.java

```diff
@@ -58,25 +58,6 @@ public void testPlainJavaProject() {
         not(MenuMatcher.hasMenuItem("Reconfigure for App Engine Java 8 runtime")));
   }
 
-  @Test
-  public void testDynamicWebProjectJava7() {
```

Comment:
This test should be kept — we should be able to convert a Java 7 web project to App Engine Java 8

Reply:
done